app/translations/cy/census_individual_gb_wls_cy.po

```diff
@@ -210,13 +214,13 @@ msgid "We are measuring the number of people who have served in the UK Armed For
 msgstr "Rydym ni’n mesur faint o bobl sydd wedi gwasanaethu yn Lluoedd Arfog y Deyrnas Unedig ac sydd bellach wedi gadael. Mae angen i’r llywodraeth a chynghorau gael y wybodaeth hon er mwyn cyflawni eu hymrwymiadau o dan Gyfamod y Lluoedd Arfog. Addewid gan y genedl yw hwn, sy'n sicrhau nad yw’r rhai sy'n gwasanaethu neu sydd wedi gwasanaethu yn y lluoedd arfog, na'u teuluoedd, o dan anfantais."
 
 msgid "Current serving members should only select “No”"
-msgstr "Aelodau sy'n gwasanaethu ar hyn o bryd, dewiswch \"Nac ydw\" yn unig"
+msgstr "Aelodau sy'n gwasanaethu ar hyn o bryd, dewiswch \"Nac ydw\"/\"Nac ydy\" yn unig"
```

Comment:
It's Welsh so not a clue but this looks dodgy?. Two lots of No?
If it is what they have done I guess that what we accept?

Reply:
The answer option is "No" in English for both proxy and non-proxy, for Welsh the answer option is "Nac Ydw" for non-proxy and "Nac Ydy" for proxy. It looks like this has been done as we don't have translation context for guidance contents. I'll check if we have a card for that and raise one if we don't.